### ISSUE TITLE
Underscore prop _interpretation returning null when using batch import

### ISSUE THREAD

**Issue by bobvanluijt** [semi-technologies/weaviate#1175]
EDIT @etiennedi: 
The issue seems to be limited to when the batch importer is used, but doesn't occur on the regular imports.

---
*Original Issue:*

To reproduce:

- Use [these Docker Compose files](http://dev.semi.technology/documentation/weaviate/current/datasets/newspublications.html#run-with-docker-compose).
- Run the following GraphQL query

```
{
  Get {
    Things {
      Article {
        _interpretation {
          source {
            concept
            occurrence
            weight
          }
        }
        summary
      }
    }
  }
}
```

Tested both locally and on the WCS.

**Comment by bobvanluijt** [comment]
note: `_classification` works correctly

**Comment by etiennedi** [comment]
Thanks for the bug ticket, investigating now. We do have a passing test, so my first suspicion would have to be that it's related to a configurable parameter (such as c11y version) where the test assumes one setting (instead of all).

**Comment by etiennedi** [comment]
Update: It works with the `0.14.0` contextionary, so something about the `0.16.0` must be broken. Since it's the same code base I can only imagine that something in the release process went wrong.

**Comment by etiennedi** [comment]
Update: It seems storing the `_interpretation` info is currently missing for the batch endpoints. It only works on the individual ones. I'll add that make a new release shortly.

**Comment by etiennedi** [comment]
fixed in `0.22.10`

**Comment by etiennedi** [comment]
![image](https://user-images.githubusercontent.com/8974479/85137449-8b6cc080-b241-11ea-85a2-95de84986b14.png)
